tools/rna_tools/vienna_rna/rna2dfold.xml

```diff
@@ -1,12 +1,14 @@
-<tool id="rna2dfold" name="RNA2Dfold" version="2.1.6.0">
+<tool id="rna2dfold" name="RNA2Dfold" version="2.2.10">
     <description>explore structure space between two reference structures</description>
-    <expand macro="requirements" />
-    <expand macro="version_command" />
-    <expand macro="stdio" />
     <macros>
         <token name="@EXECUTABLE@">RNA2Dfold</token>
         <import>macros.xml</import>
     </macros>
+    <expand macro="requirements" />
+    <expand macro="stdio" />
+    <expand macro="version_command" />
+    
+
     <command>
 <![CDATA[
         RNA2Dfold -T $temperature -d $dangling --stochBT=$nrbt
```

Comment:
Here you could use `@EXECUTABLE@ -T`
Not sure if it is so useful, but you could :)